hw_Kutovyi.py

```diff
@@ -0,0 +1,138 @@
+from typing import List
+from pydantic import BaseModel
+
+
+class BookModel(BaseModel):
+    title: str
+    author: str
+    year: int
+
+    def __str__(self):
+        return f"{self.title} ({self.author}, {self.year})"
+
+
+
+class Library:
+    def __init__(self):
+        self.books: List[BookModel] = []
+
+    def add_book(self, book: BookModel):
```

Comment:
```suggestion
    def add_book(self, book: BookModel) -> list[BookModel]:
```
And same type annotations for all method which returns smth except None